apps/backend-sls/test/integration.test.ts

```diff
@@ -1,13 +1,241 @@
+import { PrismaClient } from '@klicker-uzh/prisma'
+import Redis from 'ioredis'
 import request from 'supertest'
+import prepareApp from '../GraphQL/app'
 
-import app from '../src/app'
+// TODO: switch to ioredis-mock
+// jest.mock('ioredis', () => Redis)
```

Comment:
This would allow us to simulate redis in-memory without having to run it in the test environment. Did not get it to work yet.